bin/helpers/prepare-run-env.sh

```diff
@@ -1,11 +1,35 @@
 #!/usr/bin/env bash
 
-setDefaultRoute() {
+source bin/helpers/functions.sh
+
+function setDefaultRoute {
     GW=$1;
     ip route del default
     ip route add default via ${GW}
 }
 
+function ensure_paths {
+    iptables_path=`which iptables`
+    if [[ ${iptables_path} == "" ]]; then
+       echo "required dependency missing: iptables"
+       return
```

Comment:
Should it be `exit 1` or some error code returned here? Now it just prints output and exit with 0 status.